### DISCUSSION TITLE
Hero image?

### DISCUSSION THREAD

**Discussion by velasco** [mmistakes/jekyll-theme-basically-basic#129] [Q&A] [answered]
Apologies in advance, but how can I activate the feature and show an Hero image on the Home-site?

**Comment by mmistakes** [comment, marked as answer]
With YAML front. Look at the [`/example` folder](https://github.com/mmistakes/jekyll-theme-basically-basic/tree/master/example) for several “examples”. This one in particular has a home page with hero image.

https://github.com/mmistakes/jekyll-theme-basically-basic/blob/master/example/index.html
